src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
```

Comment:
In involve your idea, then we'll end with chttpd_blacklist_global_handlers, chttpd_blacklist_database_handlers, chttpd_blacklist_design_handlers ala httpd defines them now. Not bad decision after all.